### PR TITLE
Deploy

### PR DESCRIPTION
**Pull request by balmas**
this change deploys all branches to 

https://alpheios-misc-dev.s3.us-east-2.amazonaws.com/alignment-editor/public/index.html

We can further restrict it by branch as needed but at this stage maybe all branches is okay?

It currently deploys the entire directory because the public dir references files in dist. It would probably be better to have a script which produces a public dir with the dist directory below it and to deploy just that, because there isn't any need to deploy all the source and node_modules.  That would be a good change to make in an upcoming PR

### CODE REVIEW

File: src/vue/align-editor/align-editor-single-block.vue [hunk 1]
```diff
@@ -7,7 +7,7 @@
       <template v-for = "token in alignTextData.tokens">
         <token
           v-if ="token.word"
-          :text-type = "textType" :text-word = "token"
+          :text-type = "textType" :text-word = "token" :key = "token.idWord"
```
Comment (irina060981) [inline src/vue/align-editor/align-editor-single-block.vue:10]: it is a tiny fix for @kirlats comment 
https://github.com/alpheios-project/alignment-editor-new/pull/10#discussion_r468583873

File: .github/workflows/workflow.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      uses: jakejarvis/s3-sync-action@master
+      with:
+        args: --acl public-read --delete
+      env: 
+        AWS_S3_BUCKET: 'alpheios-misc-dev'
+        AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        AWS_REGION: 'us-east-2'
+        SOURCE_DIR: '.'
+        DEST_DIR: 'alignment-editor/public'
```
Comment (irina060981) [inline .github/workflows/workflow.yml:28]: changed to public dir

**Comment by irina060981** [comment]
@balmas, I have updated build with placing dist inside public folder.
And changed workflow.yml to upload only public

File: .github/workflows/workflow.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    - name: s3
+      uses: jakejarvis/s3-sync-action@master
+      with:
+        args: --acl public-read --delete
+      env: 
+        AWS_S3_BUCKET: 'alpheios-misc-dev'
+        AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        AWS_REGION: 'us-east-2'
+        SOURCE_DIR: 'public'
```
Comment (irina060981) [inline .github/workflows/workflow.yml:27]: changed to public dir

**Comment by balmas** [comment]
> @balmas, I have updated build with placing dist inside public folder.
> And changed workflow.yml to upload only public
> 
> Bridget, could you check? I am not sure - if I do everything right.

Yep, looks good. You can see that it deployed at https://alpheios-misc-dev.s3.us-east-2.amazonaws.com/alignment-editor/index.html

(You can see the results of GitHub Actions on the Actions tab)

**Comment by irina060981** [comment]
great :) than I believe I could merge it our joint work :)

**Pull request merged**